src/main/java/by/onliner/test_dev/transformer/AnnotationTransformer.java

```diff
@@ -1,7 +1,10 @@
-package by.onliner.retry;
+package by.onliner.test_dev.transformer;
```

Comment:
What other classes can be stored in this package? what's the point in creating a package just for one class?

Reply:
Annotation transformer is also a listener so can be stored with other listeners

Reply:
Refactor a lot of packages in "core"